### PR TITLE
Typos fixes

### PR DESCRIPTION
**Pull request by isf63**
Typos fixes to 39 files. Text/comments and identifiers corrections.

### CODE REVIEW

**Comment by yan12125** [comment]
Builds fine, thanks!

File: plugin-dom/treewindow.cpp [hunk 1]
```diff
@@ -130,7 +130,7 @@ void TreeWindow::updatePropertiesView()
     ui->propertiesView->item(PROP_OBJECT_NAME, 1)->setText(treeItem->widgetObjectName());
     ui->propertiesView->item(PROP_CLASS_NAME, 1)->setText(treeItem->widgetClassName());
     ui->propertiesView->item(PROP_TEXT, 1)->setText(treeItem->widgetText());
-    ui->propertiesView->item(PROP_CLASS_HIERARCY, 1)->setText(treeItem->widgetClassHierarcy().join(QStringLiteral(" :: ")));
+    ui->propertiesView->item(PROP_CLASS_HIERARCHY, 1)->setText(treeItem->widgetClassHierarcy().join(QStringLiteral(" :: ")));
```
[the remark this answers is not in the document]
Comment (tsujan) [reply plugin-dom/treewindow.cpp:133]: @waebbl, thanks!

@isf63, would you please fix this?

[the remark this answers is not in the document]
Comment (tsujan) [reply plugin-dom/treewindow.cpp:133]: > PR created.

Merged. Thanks.

[the remark this answers is not in the document]
Comment (waebbl) [reply plugin-dom/treewindow.cpp:133]: Thanks for the quick fix.

**Pull request merged**